operations/mimir-mixin/dashboards/writes-resources.libsonnet

```diff
@@ -5,29 +5,47 @@ local filename = 'mimir-writes-resources.json';
   [filename]:
     ($.dashboard('Writes resources') + { uid: std.md5(filename) })
     .addClusterSelectorTemplates(false)
+    .addRow(
+      $.row('Summary')
+      .addPanel(
+        $.panel('CPU') +
+        $.queryPanel($.resourceUtilizationQuery('cpu', $._config.instance_names.write, $._config.container_names.write), '{{%s}}' % $._config.per_instance_label) +
```

Comment:
Is my understanding correct that this will display a line for each pod?

Reply:
Also, why isn't this using `containerCPUUsagePanel` and instead it defines the panel itself?

Reply:
> Is my understanding correct that this will display a line for each pod?

Correct. This hasn't changed. It was even before.

Reply:
> Also, why isn't this using containerCPUUsagePanel and instead it defines the panel itself?

Because `containerCPUUsagePanel()` also display CPU request/limit (if any) but here we pass `write` matcher which could match multiple deployments in microservices mode.

Reply:
> > Also, why isn't this using containerCPUUsagePanel and instead it defines the panel itself?
> 
> Because `containerCPUUsagePanel()` also display CPU request/limit (if any) but here we pass `write` matcher which could match multiple deployments in microservices mode.

I missed this, sorry